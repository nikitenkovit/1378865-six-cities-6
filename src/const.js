export const OfferType = {
  APARTMENT: `Apartment`,
  ROOM: `Private Room`,
  HOUSE: `House`,
  HOTEL: `Hotel`
};

export const PathValue = {
  MAIN_SCREEN: `/`,
  FAVORITES_SCREEN: `/favorites`,
  ROOM_SCREEN: `/offer/:id`
};

export const CityCoordinate = {
  AMSTERDAM: {
    latitude: 52.38333,
    longitude: 4.9,
    zoom: 12
  }
}; // скорее всего временно

export const OffersListClassName = {
  CITY_PLACES: `cities__places-list`,
  NEAR_PLACES: `near-places__list`
};

export const BookmarkButtonProperty = {
  ROOM_SCREEN: {
    buttonClass: `property__bookmark-button`,
    iconClass: `property__bookmark-icon`,
    iconWidth: `31`,
    iconHeight: `33`
  },
  OFFER_CARD: {
    buttonClass: `place-card__bookmark-button`,
    iconClass: `place-card__bookmark-icon`,
    iconWidth: `18`,
    iconHeight: `19`
  }
};
