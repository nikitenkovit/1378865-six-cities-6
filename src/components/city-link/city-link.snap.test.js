import React from "react";
import {render} from "@testing-library/react";
import CityLink from "./city-link";
import {DEFAULT_CURRENT_CITY} from "../../const";

const mock = {
  city: DEFAULT_CURRENT_CITY,
  isActive: true,
  onCityLinkClick: jest.fn()
};

it(`Should CityLink render correctly`, () => {
  const {container} = render(
      <CityLink
        city={mock.city}
        isActive={mock.isActive}
        onCityLinkClick={mock.onCityLinkClick}
      />);
  expect(container).toMatchSnapshot();
});
