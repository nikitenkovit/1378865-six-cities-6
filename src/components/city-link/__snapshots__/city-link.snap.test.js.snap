// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should CityLink render correctly 1`] = `
<div>
  <a
    class="locations__item-link tabs__item tabs__item--active"
    href="#"
  >
    <span>
      Paris
    </span>
  </a>
</div>
`;
